pages/join.html

```diff
@@ -5,46 +5,29 @@
 home: false
 ---
 
-<h2>Join content goes here.</h2>
-
-<div class="contentContainer">
-  <div class="cardsContainer">
```

Comment:
It looks like this section was from a previous [ticket](https://trello.com/c/VuBaiJvq/113-join-us-project-cadence) and I think it goes above the section this PR is for. We should keep it in.